services/core-web/src/components/dashboard/customHomePage/VarianceTable.js

```diff
@@ -38,7 +38,7 @@ const defaultProps = {
 };
 
 export const VarianceTable = (props) => (
-  <div className="tab__content">
+  <React.Fragment>
     <MineVarianceTable
```

Comment:
Since removing the class, Does this still have the same margins as the other table views?

Reply:
Yes - the outer component is already adding this class so it was wrapped in two tab__contents. Now it is the same as the NoW "HomePage".

Reply:
Oh perfect.